packaging/scripts/port-occupied.c

```diff
@@ -0,0 +1,141 @@
+/* Copyright (C) 2014-2020 Daniel Dressler, Till Kamppeter, and contributors
```

Comment:
This isn't exactly a script, and should probably be in a separate repo (you could probably create a cups-container repository or something to hold all these helper programs) rather than putting it under the packaging directory.

Reply:
@rudra-iitm See above with `cups-proxyd`.